applications/Octan_Network.md

```diff
@@ -0,0 +1,181 @@
+- Octan reputation & onchain analysis reports: https://octan.network/blog/
+- Octan documentation on reputation ranking, onchain analytics, soulbound and Web3 Identity: https://docs.octan.network/octan-docs-en/overview/welcome-to-octan-network
+
+## Development Roadmap :nut_and_bolt:
+Herein, we focus on constructing a roadmap of developing Octan Reputation Ranking into the Polkadot Ecosystem, more explicitly, implementing & deploying Reputation Ranking & Soulbound on Moonbeam & Moonriver (an EVM-parachain of Polkadot Eco).  
+
+> :exclamation: If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  6 (implying 18 man months throughout the project's duration) 
+- **Total Costs:** $30,000 USD.
+
+### Milestone 1: data pipeline, Soulbound on EVM-parachains of Polkadot and onchain Reputation Ranking for Frontier/EVM-parachains of Polkadot
+
+- **Estimated duration:** 2 months
+- **FTE:**  12
+- **Costs:** $20,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains the onchain reputation concept and how a user can use the reputation board. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains the concept of onchain reputation and the vision of Octan Network. |
+| 1a. | Crawling raw data | We will build module for indexing onchain transactions, extrinsics, events from archive node |
+| 1b. | Data Warehouse slot | We will setup data warehouse slot and structure for query onchain performance related metrics |
+| 2a. | Reputation ranking algorithms | We will complete and publish a (scientific) research article about onchain reputation ranking algorithms on blockchain-based transaction datasets. |
```

Comment:
Why is "scientific" in brackets? What does "complete" mean? Published in a peer-reviewed medium? I advise against that, as you probably know peer review can take a very long time.